### DISCUSSION TITLE
我的特供板jellyfin不能安装最新版本的插件

### DISCUSSION THREAD

**Discussion by homecom2233** [metatube-community/jellyfin-plugin-metatube#364] [Q&A]
我的特供板jellyfin不能安装最新版本的插件，最高只允许到2024.205.435.0版本，更高版本的选项没有。或者能否提供一个手动安装的方法？
![QQ20240709-143719](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/37613354/ea948d51-4271-4fc1-9284-775899ef6412)

**Comment by yishisanren** [comment]
我的也是，有找到解决办法么

**Comment by yishisanren** [comment]
换一个 jellyfin 版本就可以了，我用了官方的 jeffyfin/jeffyfin最新版就可以了。
